src/activities/content/ContentScormActivityEntity.js

```diff
@@ -21,6 +22,28 @@ export class ContentScormActivityEntity extends ContentEntity {
 		return this._entity && this._entity.properties && this._entity.properties.contentServiceScormActivityUpdatedAt;
 	}
 
+	/**
+	 * @returns {string|null} HTML description of the SCORM activity
+	 */
+	descriptionRichText() {
+		const descriptionSubEntity = ContentHelperFunctions.getDescriptionSubEntity(this._entity);
+		if (!descriptionSubEntity) {
+			return null;
+		}
+		return descriptionSubEntity.properties.html || '';
```

Comment:
Is empty string the expected response when there is no html property? It seems like we are returning null above if the subentity cannot be retrieved; does the calling code handle nulls and empty strings differently?

Reply:
This was copy-pasta'd from how the weblinks entity deals with description. I can take a peek into if `null` is treated different from `''` by consumers of these entities, but I doubt it since they're both falsy.

Reply:
Drilled down and found this line in [d2l-activity-html-new-editor](https://github.com/BrightspaceHypermediaComponents/activities/blob/master/components/d2l-activity-editor/d2l-activity-html-new-editor.js#L47):
`this.value = this.value || '';`
Turns out it'll end up as `''` in the description editor itself whether it's `null` or `''` in the first place lol

Reply:
Womp womp 🤷‍♀️ . All good then I guess! Thanks for satisfying my curiosity!